### PR TITLE
Block's hue analyzer with exponential averaging added

### PR DESCRIPTION
**Pull request by CICS-Oleg**

### CODE REVIEW

File: experiments/get_blocks_hue_stats_from_user.py [hunk 1]
```diff
@@ -53,13 +55,17 @@ def _getLocalHue(self):
         avg = numpy.mean(crop_img)
```
Comment (noskill) [inline experiments/get_blocks_hue_stats_from_user.py:55]: Shouldn't it be per channel mean?

File: examples/get_recipes_from_minecraft_data.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import math
```
Comment (noskill) [inline examples/get_recipes_from_minecraft_data.py:1]: is this file a copy of  experiments/get_recipes_from_minecraft_data.py  or is it simlink?

Comment (CICS-Oleg) [reply examples/get_recipes_from_minecraft_data.py:1]: Simlink

File: experiments/get_recipes_from_minecraft_data.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import minecraft_data
```
Comment (Necr0x0Der) [inline experiments/get_recipes_from_minecraft_data.py:1]: Why do we have `examples/get_recipes_from_minecraft_data.py` both in `examples` and `experiments`?

File: experiments/get_blocks_hue_stats_from_user.py (hunk 1 above)
Comment (CICS-Oleg) [reply experiments/get_blocks_hue_stats_from_user.py:55]: I think I didn't get you right.

File: experiments/get_recipes_from_minecraft_data.py (hunk 3 above)
Comment (Necr0x0Der) [reply experiments/get_recipes_from_minecraft_data.py:1]: I still see two of them in this PR.

**Comment by noskill** [comment]
Oleg said he made a simlink. I don't know if it's necessary

вт, 18 янв. 2022 г. в 23:11, Alexey Potapov ***@***.***>:

**Comment by Necr0x0Der** [comment]
Ah, so this question about simlink was about this file. Well, I think we already have the simlink for the whole folder, so the simlink for the separate file is not needed and confusing.

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**